resource/chatadapter.go

```diff
@@ -25,7 +25,8 @@ func sendToChat(
 	team := env.Get("BUILD_TEAM_NAME")
 	buildN := env.Get("BUILD_NAME")
 	instanceVars := env.Get("BUILD_PIPELINE_INSTANCE_VARS")
-	buildURL := concourseBuildURL(atc, team, pipeline, job, buildN, instanceVars)
```

Comment:
This is the chatadapter, not yet integrated. This change is only to be able to compile and get rid of GH static analysis.